src/components/Button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { ButtonStyle } from "./Button.styled";
+interface props {
```

Comment:
Extract interfaces to a new file called types or interfaces and import these from there.